SETUP.md

```diff
@@ -1,4 +1,42 @@
-My App Setup Instructions
+Lansweeper Setup Instructions
 ===
 
-TODO: Write setup instructions
+To install the Lansweeper App, you must first create an app with a "Client ID" and "Client Secret".
```

Comment:
This doesn't say where. I assume this is in Deskpro.

Reply:
Added a sentence.